scripts/bootstrap-plugin

```diff
@@ -21,8 +21,6 @@ if [ -d $pluginname ]; then
   err "There is already a plugin with that name. Cowardly refusing to overwrite it."
 fi
 
-buildConfigFilename=${pluginname}/build.config.js
-
 mvn archetype:generate -DarchetypeGroupId=org.graylog -DarchetypeArtifactId=graylog-plugin-archetype -DartifactId=$pluginname
 
-sed 's,\(../graylog2-server/graylog2-web-interface\),../\1/,' < ${buildConfigFilename}.sample > ${buildConfigFilename}
+echo ${pluginname}/ >> ${prefix}/.gitignore
```

Comment:
Ah, okay. The we misunderstood each other, or rather I got the `bootstrap-plugin` script wrong. My suggestion was to move plugins **and** server into `modules/`. :smiley: